src/docs/components/properties.md

```diff
@@ -4,15 +4,41 @@ description: Properties
 url: /docs/properties
 contributors:
   - jthoms1
+  - rwaskiewicz
 ---
 
 # Prop Decorator
 
 Props are custom attribute/properties exposed publicly on the element that developers can provide values for. Children components should not know about or reference parent components, so Props should be used to pass data down from the parent to the child. Components need to explicitly declare the Props they expect to receive using the `@Prop()` decorator. Props can be a `number`, `string`, `boolean`, or even an `Object` or `Array`. By default, when a member decorated with a `@Prop()` decorator is set, the component will efficiently rerender.
 
 ```tsx
+// TodoList.tsx
 import { Prop } from '@stencil/core';
+import { MyHttpService } from '../some/local/directory/MyHttpService';
+...
+export class TodoList {
+  @Prop() color: string;
+  @Prop() favoriteNumber: number;
+  @Prop() isSelected: boolean;
+  @Prop() myHttpService: MyHttpService;
+}
+```
+
+When using user-defined types like `MyHttpService`, the type must be exported using the `export` keyword. Above,
+`MyHttpService` is imported from `'../some/local/directory/MyHttpService'`, and must be exported from that file 
+(otherwise we'd have no way to use it).
```

Comment:
I don't think you need the line in parens. I think your explanation is good.

Reply:
Done! https://github.com/ionic-team/stencil-site/pull/784/commits/bec3cc5b3159cf1277aa5af7b2f87aaa4bacbb5a